sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -29,7 +29,7 @@ IUSE="examples"
 # If you are building from git, you will also need gperf to generate
 # the configure scripts.
 RDEPEND="
-	sys-libs/readline:0
+	sys-libs/readline:=
```

Comment:
This change requires a revbump. You can use `git mv` here.

Reply:
> This change requires a revbump. You can use `git mv` here.

Thanks for reminding, because it have stable keywords, I do a revbump and keep the stable ebuild. When this revbump version stable, I'll drop the stable version.